lib/engine/game/g_1817.rb

```diff
@@ -459,6 +459,15 @@ def total_rounds(name)
         @operating_rounds if ['Operating', 'Merger', 'Merger and Conversion', 'Acquisition'].include?(name)
       end
 
+      def corporation_size(entity)
+        # For display purposes is a corporation small, medium or large
+        { 2 => :small, 5 => :medium, 10 => :large }[entity.total_shares]
```

Comment:
Should this hash be a constant on the class so it only gets allocated once per game?